src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/xpath/regex/RegularExpression.java

```diff
@@ -473,7 +473,7 @@
  * <hr width="50%">
  * <h3>TODO</h3>
  * <ul>
- *   <li><a href="http://www.unicode.org/unicode/reports/tr18/">Unicode Regular Expression Guidelines</a>
+ *   <li><a href="http://www.unicode.org/reports/tr18/">Unicode Regular Expression Guidelines</a>
```

Comment:
Just a note that the references had been report numbers, e.g. Report/Annex #15, while this one the Title (by the way, the official title was "Unicode Regular Expressions"). The only exception seems to be the one in NormalizerBase.java where it referred to the number + title.